dragon_gigapet/Gigapet.rb

```diff
@@ -0,0 +1,58 @@
+#!/bin/end ruby
+require_relative 'Animal'
+require_relative 'Dragon'
+
+
+puts "_" * 26
+puts "XXXX |Dragon Gigapet| XXXX"
+puts "-" * 26
+
+
+
+puts " You just cleared the dunegeon. There seems to be a chest near the exit. You open the chest to find a Dragon's egg."
+puts " Your DM ask for a d10 roll"
+puts ">>>"
+
+
+
+
+roll = gets.chomp.to_i
```

Comment:
Extra credit question when you finish the project 😛  What happens if I type in jibberish here? Or nothing? Same for the color input below.

Reply:
hahaha, it would. assign that said gibberish to roll. So I need to figure out a way for gets to only accept certain input